src/main/java/com/exadel/ipromise/dao/impl/PromiseDao.java

```diff
@@ -0,0 +1,43 @@
+package com.exadel.ipromise.dao.impl;
+
+import com.exadel.ipromise.entity.Promise;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+
+@Repository
+public class PromiseDao {
```

Comment:
use interface implementation. Because in case we have free time we will implement  Spring Data instead JDBC and in case you use interface instead directly object - it will be easy to switch into another interface implementation.

Reply:
Oh no, I didn't create an interface. Thank you. 😊

Reply:
Done